signer-npm/src/lib.rs

```diff
@@ -984,14 +984,28 @@ pub fn verify_voucher_signature(
 }
 
 #[wasm_bindgen(js_name = getCid)]
-pub fn get_cid(signed_message: JsValue) -> Result<String, JsValue> {
+pub fn get_cid(message: JsValue) -> Result<String, JsValue> {
     set_panic_hook();
 
-    let signed_message_api: SignedMessageAPI = signed_message
+    let message_api: MessageTxAPI = message
         .into_serde()
         .map_err(|e| JsValue::from(format!("Error parsing parameters: {}", e)))?;
 
-    let result = filecoin_signer::get_cid(signed_message_api)
+    let result = filecoin_signer::get_cid(message_api.get_message())
+        .map_err(|e| JsValue::from(format!("Error getting the cid: {}", e)))?;
+
+    Ok(result)
+}
+
+#[wasm_bindgen(js_name = getBlsSigningBytes)]
```

Comment:
In Filecoin, the signing bytes are actually the CID so if we can manage to support unsigned message in the `getCID` function we wouldn't need that function. Sorry it is a bit confusing it should have been mentioned in the issue as there would be no way for you to know.

Reply:
Hmm since it's only the CID I assume the other signature type also needs to change was is signed?

I'll delete this function anyways, since it's a duplicate of `get_cid`

Reply:
I tried looking into the other signature type but the procedure is completely different (to retrieve the `blob_to_sign`), since it hashes the message and the CID before signing...

How should I proceed? Only use the CID from the `UnsignedMessage` ?

Reply:
You can use `to_signing_bytes` and if you look at the implementation it is returning the CID of the `UnsignedMessage` as bytes.

https://docs.rs/crate/forest_message/0.6.0/source/src/unsigned_message.rs

Reply:
But we don't need this function right. I believe you remove this.

Reply:
Yes I did remove this function but the question is another:
We have 2 types of signature (as encoded in the `Signature` enum): BLS and SECP256K1.
I changed how the BLS signature is made, but not the other one. 
From the spec it seems like they should be made the same way, signing the CID for the message instead of the whole message, but I am not able to understand how to construct the SECP256K1 signature correctly just by looking at the code of the current method.
So, how do I create the SECP256K1 signature correctly?